src/scripts/chart/column.js

```diff
@@ -49,7 +61,7 @@ function columnChart(g){
 	'use strict';
 
 	var model = new DataModel(Object.create(g.data()[0]));
-	var i = model.y.series.length;
+	var i = 0;
```

Comment:
var i = 0 is redundant as it's defined below on line 94 as part of the for loop.